app/presenters/root_browse_page_presenter.rb

```diff
@@ -0,0 +1,30 @@
+class RootBrowsePagePresenter
+
+  def render_for_publishing_api
+    {
+      content_id: "8413047e-570a-448b-b8cb-d288a12807dd",
+      format: "mainstream_browse_page",
+      title: "Browse",
+      locale: 'en',
+      public_updated_at: Time.now.iso8601,
```

Comment:
I understand, and now it make a lot of sense, thanks!

What do you think if I add a method that runs through the top-level pages, finds the most recent and sticks its updated_at time in here? 
I know this is not going to happen in a real world scenario but, what should the method return in case of total absence of top-level pages? `Time.now.iso8601`, `nil` ?